spec/liquid_tags/tweet_tag_spec.rb

```diff
@@ -23,11 +23,14 @@ def generate_tweet_liquid_tag(id)
     end
   end
 
-  xit "render properly", :vcr do
+  it "render properly", :vcr do
```

Comment:
This was a skipped test previously, but now that I've "un-`approvals`"ed it, I think we can unskip it :) 

It seems to consistently pass for me, but if it becomes flaky again or if we need to skip it again down the road, I figured we can cross that bridge when we come to it.

Reply:
agreed! left a non blocking suggestion

```suggestion
  it "renders properly", :vcr do
```